lib/gui/app/modules/analytics.js

```diff
@@ -70,14 +76,8 @@ function getConfig(url) {
  * @param {Object} - config
  */
 // eslint-disable-next-line
-function getMixpanelConfig(config) {
-  const analytics = config.analytics || {}
-  const mixpanel = analytics.mixpanel || {}
-  const probability = mixpanel.probability || DEFAULT_PROBABILITY
-  if (Math.random() > probability) {
-    return null
-  }
-  return validateMixpanelConfig(mixpanel)
+function isClientEligible(probability) {
```

Comment:
You can use `mixpanelSample` and remove the parameter.

Reply:
I don't like the idea of the function depending on a parameter outside of its context tbh